### ISSUE TITLE
Enhancement: Gmail does not issue notifications when receiving Google Hangouts messages

### ISSUE THREAD

**Issue by moghingold** [TheGoddessInari/rambox#8]
### Steps to reproduce

1. Add a Rambox service using the Gmail preset
2. Sign into Gmail
3. Navigate away from the Gmail tab within Rambox, or away from Rambox in your OS session
4. Receive a Google Hangout while signed into Gmail in Rambox

### Expected behavior

The number of notifications in the Gmail tab in Rambox should increment by the number of Hangouts conversations with unread messages

### Actual behavior

Nothing

### Justification

Because Google Hangouts is integrated into Gmail, it would be very convenient if unread Hangouts produced the same behavior as unread emails.

**Comment by TheGoddessInari** [comment]
I would be happy to add it, but Google has a habit of making its products hard to parse.
If anyone can give me the equivalent function or DOM information, I'd be happy to merge it.
